### DISCUSSION TITLE
求激活，大佬

### DISCUSSION THREAD

**Discussion by wzjeh** [LFTEC/LearningFucker#49] [General]
NUH0fzHWXklDnvuULCv31ie5NgMNglGQAbISDS5Ixv78m0hA/7lGhsw3MNCoBD5tDBL
OWhOD9U0MTio2U/WMvHznKfes1lAuoaJTOKnZ4SYQfmrXbqUiA/cVMCZJSg6bOxK6Bl
Ue6q5AVpk6XtK1V9OmNfW3gxZuzb+irMdTaaA6qXfkD48iE+5/hKRqsV5nD1YtPLhu1
4QSOkHPPLZNeo4pt+8tijnenqwwGRl2GAX+fkP/EpUeFUjAm8KLn2noBOgwumOyAGIX
TtyKIyWX0kops5omaIP/ta9+fCiFLqNU4e30uLdQJ2sTFbMF454FvvE3KzoBAVXx0gg
zPvOm2znhyPIEL0PRT+WRElsGVTs6CTp5fsxnCeC4deHDO/l7oeTH3lkQFT7KPXGSQU
NH8AtlPp7oh1MK0zHtAsStCF6GOgmBdRj0BZIsuzdGLvwRLQOGUhf15eGNIjYr8JJ7o
0KsWXaifJjxvSk2CfCprWd07lc+3C8eOOdHgN3PCaghW7/ujdnljxWzrFZe5FJLmQ/3
6XBEN899vxnSnFfLDhG69FySHfPpDI3l7W41t7/TqmYwm9EYKOfkmX7cvh56DNhhdrS
wo/KpmF2vzQ6EBW5scIUG8DJYiRFJAVlxZPQuTZUrxp3rvwgv2uBkSaJ6/Lc1mLExVI
5lSkc3H8hrw/OxlM4BDGJ+qR2ZbdzvhQljeYhrf9jpy/E3MFYvgwnupd6Fxao0sFjV9
vZl4S7YcfSb81aRJhVTKFtpuP5pGUfuLAZEvA6tzydQS4Iwn/d74zf+baK6bT5HyhV3
DvVEszHdLQ2ntuetdJ3w6XD6BO69cyFqC6e2xIOL5leAONRPv2nA8OyFak1xY6J/0Nn
+U/4nVI6NBpQNiiEh9G10dseETxnXW1MnTmSNX5CAX2utmz6kRTFmOavz0p0aIcdccv
QWk19m1eocuRq4Qs9XUYd6FLrvJvtaqdt1qE5POaO+/NTqQPQhXOnBh+RjpfaXIyYS1
7hF+vNyXWM/ZEVzZCNvocFqkrKrhKG4eZ4rORAuGARJepTtwerRVfIld9Hmn+LqQiTi
u9yEGgKzLfN0wX5mvHlrj8jU66NKGVErp4IeTACJabwGVD0XtjEQnpPH1qM6dPtOZyx
pBaz1fyEHr9kusHqotiGqjna7fLYR8CdSZPka51zVOnN6F/zuwaWPWxUkNBJGu0M40S
pKUQgTGwyqRagaReClCDCaF90w3jhKLJ/dteLFlOm0YkBYT0nPwS0gysICMLvRuxY/R
BHPARYL/YGr5kr+r6tuSiso27P3M17bXEijMm/iJDq3oamI67pgNDMeiOqTsmEIr7Y8
O/uAD+ocVcNU6R7QOp8jNqHJ/Q==

**Comment by LFTEC** [comment]
n/BQspzIsK6juW1uwtbp84r3tV0aImGWBwsfxqr7nH0bSsBl3utUvu7ZMBxct38s0XGqxzXDANXXzpYhlJ1XXON0H/ljhnTD4/JoVCI3poCGprzCfxMGLfYyb+7UghpkDttMMqy6H7Vtq0rIPXvXIRnpEZ0kBTgQr1ksx8vmuKVuI5HgyGT5XXQ8NyMVuguyaNf9RTQc7U4hOB/SUFnO4NiYlnmQomGJ9UmySaVsQAqNoaOqOSHb3GgrKKR3BQwZgyVM5yLHpngsBI7TRddaloZBM+nneF7Lh+DMGli0J1hExaPFqDGUXx2/QD16U8Qn8fRBH4SO4220hN4YmUhYGTzxGYKZzG74PfGrNhOY6LIVT3a4jG2ZbQ6PWF6YzLbIuvDXuTxiqI7HUC1A415wQSxrqMzTZ87Tw/UlnV/feEGlIBYFCdV2w2UJKAlIMAkIpw/cS8+DqSevUG0gtgXIUee+XC4xbqom6YgvDwI3htM/+nNU4oGo8iyuEatMC6DzVwSeJyIpxmXxRYouX/geLji0KnEDw4JI7FKX61nbfQCmHXfPxIZ+LZ3w3sQdi7UqwhzxmJ3XhReSOAXNj3EgwiNJZFeo7j/lTHb6JxSbGytVgGJVgQUf9DKXJKy/teDxF6x6KksdCxteN53BQ5bkFKwJrAOk1e16V0YnWe3jBoaCcvuGcjBcJj0y3gdWcK5yf2SPhAH4TTDz42m308h6x6f1GgrmZCAK3g/jwVj/M+9SPfsbaEOYXlP447BnGJcnNsu2EFWDLLfsl6R0o7I7LZYMq4QgP9GjSZD1U/6Ov04IgINT2S41UBuCaaqy2HhrTdhXJFmCfG0SV7f2pk0F2dkPKiNa/yyslPPFS11hEBLZkS4TRLO6tz864XB4mYNXvqcRojkDNmBe4AmwwsmKJfaIzX9V+iOVJ2PDoootCdBY8H3sBRFp/Eb1TNBovw8a9MaIMroN+L+KkiR+WyJuHRlyQSmVSwN4s1W/olKx9/YZN9EiRwMZNSnpp/wWEEXDht2bOnng0la2+eNn4nyQvOAI8VcvhnOXaFK3pNN8lYtmPpb33gtAGulArwQxTSGtgcFpe01QF1Gg72pnZ63/HNPN/1cPPD4nBsoG+mncFPmUcm3XaB0j+dBpn4dJNlpO6OXDXEPunp7QQJPiRan0SP+CZPUHHB4KCrZag45DxVW1E432E1NYHjDR/uejzODRb/irs0BPgYnAa9ti+9dmw0NFvZvSjGudVQWWh1BKRk00fP2HhbvouKWLZpIt5xiv0TkSeft5wbWTR1LjN9MVr1i+nEJphrgrFfa4NW8rqQbQ52XVHZ1o3gphuYbuj3SrmP7vNFYYawaZqOn6EiwGLQ==
